detection_rules/etc/stack-schema-map.yaml

```diff
@@ -122,4 +122,9 @@
 "8.18.0":
   beats: "8.17.1"
   ecs: "8.17.0"
+  endgame: "8.4.0"
+
+"9.0.0":
+  beats: "9.0.0-beta1"
```

Comment:
Note, the beta and rc packages are abnormal and not recommended. However, we have little choice as there are not full release packages available.